vernac/vernacentries.ml

```diff
@@ -2195,6 +2191,7 @@ let with_fail st b f =
   end
 
 let interp ?(verbosely=true) ?proof ~st (loc,c) =
+  let orig_univ_poly = Flags.is_universe_polymorphism () in
   let orig_program_mode = Flags.is_program_mode () in
   let rec aux ?polymorphism ~atts isprogcmd = function
```

Comment:
Should `polymorphism` go in the `atts`?

Reply:
Note quite. IIUC, `polymorphism` is the value written explicitly by the user (a `bool option`). In `atts`, you find the computed value (a `bool`).

Reply:
The distinction is "useful" to make sure polymorphism is not specified twice. @vbgl is probably going to get rid of this redundancy in his attribute branch.